common/guild_base.cpp

```diff
@@ -866,20 +866,11 @@ bool BaseGuildManager::QueryWithLogging(std::string query, const char *errmsg) {
 	return(true);
 }
 
-//factored out so I dont have to copy this crap.
-#ifdef BOTS
 #define GuildMemberBaseQuery \
```

Comment:
This should still if/else considering this relies on views and different tables depending on whether bots are enabled or not

We need to eliminate these views in code and need to be substituted for queries